FinniversKit/Sources/Components/JobKeyInfo/JobKeyInfoView.swift

```diff
@@ -0,0 +1,34 @@
+import UIKit
+
+public class JobKeyInfoView: UIView {
+    public typealias InfoPair = (title: String, value: String)
+
+    // MARK: - Private properties
+
+    private lazy var stackView = UIStackView(axis: .vertical, spacing: .spacingS, withAutoLayout: true)
+
+    // MARK: - Init
+
+    public init(infoPairs: [InfoPair], withAutoLayout: Bool) {
+        super.init(frame: .zero)
+        translatesAutoresizingMaskIntoConstraints = !withAutoLayout
+        setup(infoPairs: infoPairs)
+    }
+
+    public required init?(coder: NSCoder) { fatalError() }
```

Comment:
Just to follow same style, but not necessary: "fatalError("init(coder:) has not been implemented")"